instrumentation/kafka/kafka-clients/kafka-clients-2.6/library/README.md

```diff
@@ -130,17 +130,17 @@ collected.
 | `consumer-fetch-manager-metrics` | `bytes-consumed-rate`                       | `client-id`,`topic`             | `kafka.consumer.bytes_consumed_rate`                       | The average number of bytes consumed per second                                                                                             | `DOUBLE_OBSERVABLE_GAUGE`   |
 | `consumer-fetch-manager-metrics` | `bytes-consumed-total`                      | `client-id`                     |                                                            |                                                                                                                                             |                             |
 | `consumer-fetch-manager-metrics` | `bytes-consumed-total`                      | `client-id`,`topic`             | `kafka.consumer.bytes_consumed_total`                      | The total number of bytes consumed                                                                                                          | `DOUBLE_OBSERVABLE_COUNTER` |
-| `consumer-fetch-manager-metrics` | `fetch-latency-avg`                         | `client-id`                     | `kafka.consumer.fetch_latency_avg`                         | The average time taken for a fetch request.                                                                                                 | `DOUBLE_OBSERVABLE_GAUGE`   |
-| `consumer-fetch-manager-metrics` | `fetch-latency-max`                         | `client-id`                     | `kafka.consumer.fetch_latency_max`                         | The max time taken for any fetch request.                                                                                                   | `DOUBLE_OBSERVABLE_GAUGE`   |
-| `consumer-fetch-manager-metrics` | `fetch-rate`                                | `client-id`                     | `kafka.consumer.fetch_rate`                                | The number of fetch requests per second.                                                                                                    | `DOUBLE_OBSERVABLE_GAUGE`   |
+| `consumer-fetch-manager-metrics` | `fetch-latency-avg`                         | `client-id`                     | `kafka.consumer.fetch_latency_avg`                         | The average time taken for a fetch request                                                                                                  | `DOUBLE_OBSERVABLE_GAUGE`   |
```

Comment:
the existing OpenTelemetry convention has been to include the trailing period, see https://github.com/open-telemetry/opentelemetry-specification/issues/2975 for motivation

Reply:
@trask I have updated the existing ones to use trailing periods

[`89997fa` (#9621)](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9621/commits/89997faeacfaa2fc1f6706ec4c826e13c76ece7a)